webextensions/api/menus.json

```diff
@@ -324,7 +324,7 @@
             "support": {
               "chrome": {
                 "alternative_name": "contextMenus.OnClickData",
-                "version_added": "≤23"
+                "version_added": "7"
```

Comment:
✅  Based on https://github.com/chromium/chromium/commit/aefa99e2b6abcb3e1b347d10d2b9a0e4110c68f7 (note: this is a doc change only), it is in [6.0.492.0](https://github.com/chromium/chromium/releases/tag/6.0.492.0), and Chrome 6 was based on 6.0.472 (_before_ that), so it landed in Chrome 7.